Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/Routing.rst

```diff
@@ -21,6 +28,32 @@ Example:
 
 
 API
-===
+---
 
 ..  include:: /CodeSnippets/Manual/Entity/PageArguments.rst.txt
+
+
+..  _typo3-request-attribute-routing-backend:
+
+Backend
+=======
+
+..  versionadded:: 12.1
+
+The :php:`routing` backend request attribute provides routing information in
+the object :php:`\TYPO3\CMS\Backend\Routing\RouteResult`.
+
+Example:
+
+..  code-block:: php
+
+    $routing = $request->getAttribute('routing');
```

Comment:
we should mention $requests type

Reply:
It is in the request chapter about request attributes. When I add it here it should also be added to every single request attribute example. This would be then a separate PR.

Reply:
There is a difference on what you can do with extbase requests and none extbase requests, see for example https://github.com/TYPO3-Documentation/TYPO3CMS-Exceptions/pull/95

So the context we assume by telling where we are very much matters I think

Reply:
Yes, but here it is all about the PSR-7 request object, as the Extbase request object implements that interface (since v11) it has the routing attribute. So no need to differentiate between these two for this case.

Reply:
How about adding `/** @var ServerRequestInterface */` then?